README.md

```diff
@@ -19,7 +19,7 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Awesome Book] <a name="about-project"></a>
+# 📖 [My Bookstore Project] <a name="about-project"></a>
```

Comment:
Great job @ClintonYounge 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.